cf/dimensioncoordinate.py

```diff
@@ -334,6 +334,106 @@ def direction(self):
 
         return _direction
 
+    @classmethod
+    def create_regular(
+        cls, args, units=None, standard_name=None, bounds=True
+    ):
+        """
+        Create a new `DimensionCoordinate` with the given range and regular
+        cellsize.
```

Comment:
PEP8 tells us to keep this to one line. How about
```suggestion
        Create a new `DimensionCoordinate` with the given range and cellsize.
```

Reply:
Hi Ankit - this, and other, suggestions are marked as reolved but are not reflected in the code - was that intended? Thanks.